components/Valuation/Graph.tsx

```diff
@@ -0,0 +1,125 @@
+      },
+      layout: {
+        backgroundColor: "#131722",
+        textColor: "#d1d4dc",
+      },
+      grid: {
+        vertLines: {
+          color: "rgba(42, 46, 57, 0)",
+        },
+        horzLines: {
+          color: "rgba(42, 46, 57, 0.6)",
+        },
+      },
+    });
+    const areaSeries = chart.addAreaSeries({
+      topColor: "rgba(38,198,218, 0.56)",
+      bottomColor: "rgba(38,198,218, 0.04)",
+      lineColor: "rgba(38,198,218, 1)",
+      lineWidth: 2,
+      title:"Daily Volume"
+    });
+
+    const volumeSeries = chart.addHistogramSeries({
+      color: "#26a69a",
+      title:"Floor Price",
+      priceFormat: {
+        type: "volume",
+      },
+      priceScaleId: "",
+      scaleMargins: {
+        top: 0.8,
+        bottom: 0,
+      },
+    });
+    areaSeries.setData(
+      values.map((data) => {
+        return {
+          time: new Date(data.time) / 1000,
+          value:
+            data.dailyVolume[
+              (symbol === "ETH" && "ethPrediction") ||
+                (symbol === "USDC" && "usdPrediction") ||
+                (symbol === "Metaverse" && "metaversePrediction")
+            ],
+        };
+      })
+    );
+    volumeSeries.setData(
+      values.map((data) => {
+        return {
+          time: new Date(data.time) / 1000,
+          value:
+            data.floorPrice[
+              (symbol === "ETH" && "ethPrediction") ||
+                (symbol === "USDC" && "usdPrediction") ||
+                (symbol === "Metaverse" && "metaversePrediction")
+            ],
+        };
+      })
+    );
+    const resizeGraph = () =>
+      chart.applyOptions({ width: metaverseGraph?.current!.clientWidth! });
+    window.addEventListener("resize", resizeGraph);
+    return () => {
+      window.removeEventListener("resize", resizeGraph);
+
+      chart.remove();
+    };
+  }, [values, symbol]);
+  return (
+    <div className="max-w-full h-full relative" ref={metaverseGraph}>
+      <div className="absolute top-1 left-1 z-10 flex gap-2">
+        <button
+          className={"gray-box font-semibold  rounded-lg p-2 text-xs text-gray-400"+(symbol==="ETH"?" text-gray-300 bg-opacity-80 ":" hover:text-gray-300 hover:bg-opacity-80")}
+          onClick={() => setSymbol("ETH")}
+        >
+          ETH
+          
+        </button>
+        <button
+          className={"gray-box font-semibold rounded-lg p-2 text-xs text-gray-400"+(symbol==="USDC"?" text-gray-300 bg-opacity-80 ":" hover:text-gray-300 hover:bg-opacity-80")}
+          onClick={() => setSymbol("USDC")}
+        >
+          USDC
+        </button>
+        <button
+          className={"gray-box font-semibold rounded-lg p-2 text-xs text-gray-400"+(symbol==="Metaverse"?" text-gray-300 bg-opacity-80 ":" hover:text-gray-300 hover:bg-opacity-80")}
+          onClick={() => setSymbol("Metaverse")}
+        >
+          {metaverse==="sandbox" && "SAND"||metaverse==="decentraland" && "MANA"||metaverse==="axie-infinity" && "AXS"}
```

Comment:
Usually, if you have more than two if statements comparing the same thing. I would recommend making an object at the top and taking the values from there.

`const metaverseOptions = {
sandbox: {name: sandbox, otherValues: etc..},
decentraland:  {name: decentraland, otherValues: etc..}
}`

Then in here you can just write `{ metaverseOptions[metaverse].name}`

This I would also apply to the buttons. If you are writing 3 buttons that do the same, try to apply the same concept and have just one button written with `onClick={() => setSymbol(object[coin])}`

and the same with the text on lines 106,113,119.  

This 3 buttons, try to just write one and use this method that I just decribed.